react/components/AttachmentList.js

```diff
@@ -1,75 +1,35 @@
-import React, { Component } from 'react'
-import { intlShape, injectIntl } from 'react-intl'
-import ProductPrice from 'vtex.store-components/ProductPrice'
+import React from 'react'
 import { productShape } from '../utils/propTypes'
-import { propOr } from 'ramda'
+import { pathOr } from 'ramda'
 
-import { optionPricePerItem, parentPricePerUnit, getProductPrice, CHOICE_TYPES } from '../utils/attachmentHelper'
+import RemovedAttachmentsList from './RemovedAttachmentsList'
+import AddedAttachmentsList from './AddedAttachmentsList'
 
-class AttachmentList extends Component {
-  static propTypes = {
-    product: productShape,
-    intl: intlShape,
-  }
+import styles from '../productSummary.css'
 
-  getOptionExtraQuantity = (option) => {
-    const { product } = this.props
-    const { quantity, compositionItem } = option
-    const currentQuantity = quantity / product.quantity
-    const initialQuantity = propOr(0, 'initialQuantity', compositionItem)
-    return currentQuantity - initialQuantity
-  }
+const canShow = option => option.extraQuantity > 0 || option.item.sellingPrice !== 0 
```

Comment:
this name could be a little more descriptive. how about `shouldShowOption`?

Reply:
or rather, since it's used in a single place, why is it an external function? it could be defined inside the `filter` from where it's called, I think it would be clearer, even.

Reply:
now I see you just moved this function around, so ok, but this suggestion stands